### DISCUSSION TITLE
change default download directory and disable download prompt when executing remote thru selenoid

### DISCUSSION THREAD

**Discussion by webmoskal** [seleniumbase/SeleniumBase#1436] [Q&A] [answered]
Trying to override default download location and not get download dialog when downloading a file using selenoid(hub). here is my cap file:
`desired_cap = {
    "browserName": "chrome",
    "browserVersion": "95.0",
    "selenoid:options": {
        "enableVNC": True,
        "enableVideo": True
    },
    "screenResolution": "1345x1835x24",
    "prefs": {
        "download.default_directory":  "/home/selenium/Downloads",
        "download.prompt_for_download": False,
        "download.directory_upgrade": True,
        "safebrowsing.enabled": False,
        "safebrowsing.disable_download_protection": True,
    }
}`

**Comment by mdmintz** [comment, marked as answer]
Grid configuration for the downloads folder location via desired capabilities is based on the options available for the specific Selenium Grid that you're using. Also, if the Grid node is not on the same machine as the Grid server, you'll run into the problem that the downloads only exist on the grid node machines where the actual browsers are spun up, as opposed to the grid server where the tests are launched. You might need a separate script to copy over the files after they have been downloaded.
